### PR TITLE
For compatbility with HTK USEPOWER option have added --use-power opti…

### PR DESCRIPTION
**Pull request by edwinferretjp**
…on (default[=true] is same as current code) to filterbank generation code. To use add --use-power=[false|true] to feature config file.

### CODE REVIEW

File: src/feat/feature-fbank.h [hunk 1]
```diff
@@ -55,7 +56,8 @@ struct FbankOptions {
                  energy_floor(0.0),  // not in log scale: a small value e.g. 1.0e-10
                  raw_energy(true),
                  htk_compat(false),
-                 use_log_fbank(true) {}
+    use_log_fbank(true),
```
Comment (danpovey) [inline src/feat/feature-fbank.h]: there is odd indentation here.  Please make sure you are writing spaces not tabs.

File: src/feat/feature-fbank.h [hunk 2]
```diff
@@ -71,6 +73,8 @@ struct FbankOptions {
                    "to change other parameters).");
     opts->Register("use-log-fbank", &use_log_fbank,
                    "If true, produce log-filterbank, else produce linear.");
+    opts->Register("use-power", &use_power,
+		   "If true, use power in filterbank analysis, else magnitude.");
```
Comment (danpovey) [inline src/feat/feature-fbank.h]: please check that this line doesn't  exceed 80 characters.  And replace tabs with spaces...

**Comment by galv** [comment]
I see you pushed a new commit. When you make a fix and you want it to be seen, you need to manually comment here. We aren't notified of commits by themselves.

**Comment by danpovey** [comment]
Thanks.. merging.

**Comment by edwinferretjp** [comment]
Thanks. Noted for the future.

**Pull request merged**